drizzlepac/haputils/which_skycell.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+"""Returns the name(s) of the projection cell(s)/skycell(s) that the observations in the user-specified input
+ file occupy."""
+
+import argparse
+import glob
+import os
+import sys
+
+from drizzlepac.haputils import cell_utils
+
+
+__version__ = 0.1
+__version_date__ = '08-Sept-2021'
+# ------------------------------------------------------------------------------------------------------------
+
+
+def identify_pc_sc(img_list):
```

Comment:
This function name seems a little...obscure.  Changing this to something a little more explicit would help the user understand what it can do; perhaps, 'report_sky_cells', where 'report' indicates it is telling the user the information.

Reply:
Issue resolved by commit [27fa35f](https://github.com/spacetelescope/drizzlepac/pull/1125/commits/27fa35ff65fb597a10a1c0d751752d5e7bd6e40c).